docs/getting_started/install_manually.md

```diff
@@ -1,29 +1,284 @@
+You can install eZ Platform manually on the following operating systems:
 
-Hi! You are about to install eZ Platform on your machine and this guide is here to make sure that the whole process of preparation and installation is fast and easy. This guide consists of a three paths that differ slightly; you should choose the one that meets your operating system:
+- [Linux](#installing-on-linux)
+- [macOS](#installing-on-macos) (*development only*)
+- [Windows](#installing-on-windows) (*development only*)
 
-[Microsoft Windows](install_manually.md#manual-installation-on-windows), [Mac OS X](install_manually.md#installation-guide-for-os-x) or other [Unix-Based Systems](install_manually.md#installation-guide-for-unix-based-systems).
+!!! caution "Supported systems"
 
-**Installation guides can be followed with any eZ Symfony distribution, you can find a list of available distributions from eZ in a table below:**
+    Only installation on Linux is fully supported.
 
-| Type                                       | Archive                                                                   | License                                         | GIT */ Composer*                                                                                                                                   |
-|--------------------------------------------|---------------------------------------------------------------------------|-------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------|
-| eZ Platform *- "clean"*                    | [ezplatform.com](https://ezplatform.com/#download-option) | GPL                                             | [ezsystems/ezplatform](https://github.com/ezsystems/ezplatform) ([INSTALL.md](https://github.com/ezsystems/ezplatform/blob/master/INSTALL.md))     |
-| eZ Platform *- "demo"*                     | *Available via Git / Composer*                                            | GPL                                             | [ezsystems/ezplatform-demo](https://github.com/ezsystems/ezplatform-demo)                                                                          |
-| eZ Platform Enterprise Edition *- "clean"* | [support.ez.no/Downloads](https://support.ez.no/Downloads)                | BUL (***requires eZ Enterprise subscription)*** | [ezsystems/ezplatform-ee](https://github.com/ezsystems/ezplatform-ee) ([INSTALL.md](https://github.com/ezsystems/ezstudio/blob/master/INSTALL.md)) |
-| eZ Platform Enterprise Edition - *"demo"*  | [support.ez.no/Downloads](https://support.ez.no/Downloads)                | BUL (***requires eZ Enterprise subscription)*** | [ezsystems/ezplatform-ee-demo](https://github.com/ezsystems/ezplatform-ee-demo)                                                                    |
+    Installations on macOS or Windows can only be used for development or experimentally.
```

Comment:
maybe: should be used for development only? 
let's not get them the idea they can experiment with Windows